tasks/components/layout/property/PropertyDisplay.tsx

```diff
@@ -0,0 +1,248 @@
+      },
+      field: {
+        fieldType: 'date',
+        entryList: ['Apple', 'Banana', 'Lemon'],
+        isAllowingCustomValues: false
+      },
+      rules: {
+        importance: 'optional',
+        isAlwaysVisible: true
+      }
+    }
+  ]
+  const [subjectType, setSubjectType] = useState<SubjectType | undefined>(initialSubjectType)
+  const [search, setSearch] = useState<string>(initialSearchValue)
+
+  // TODO could be computationally intensive consider forwarding to the backend later on
+  const filteredProperties = MultiSubjectSearchWithMapping([subjectType ?? '', search], propertyList,
+    property => [property.basicInfo.propertyName, property.basicInfo.description, property.basicInfo.subjectType])
+
+  return (
+    <div className={tw('py-4 px-6 flex flex-col gap-y-4')}>
+      <div className={tw('flex flex-row gap-x-1 items-center')}>
+        <Tag className={tw('text-hw-primary-400')} size={20}/>
+        <Span type="title">{translation.properties}</Span>
+      </div>
+      <div className={tw('flex flex-col gap-y-2')}>
+        <div className={tw('flex flex-row justify-between')}>
+          <div className={tw('flex flex-row gap-x-2')}>
+            <Input
+              // TODO Search Icon
+              value={search}
+              onChange={setSearch}
+              onEditCompleted={setSearch}
+              placeholder={translation.search}
+            />
+            <PropertySubjectTypeSelect
+              className={tw('w-full')}
+              value={subjectType}
+              onChange={setSubjectType}
+              hintText={translation.subjectType}
+            />
+            <Button
+              className={tw('w-full !px-0')}
+              variant="textButton"
+              color="negative"
+              onClick={() => {
+                setSubjectType(undefined)
+                setSearch('')
+              }}
+            >
+              {translation.removeFilter}
+            </Button>
+          </div>
+          <Button onClick={() => updateContext({
+            ...contextState,
+            propertyId: undefined
+          })}>
+            <div className={tw('flex flex-row gap-x-2 items-center')}>
+              <Plus/>
+              <Span>{translation.addProperty}</Span>
+            </div>
+          </Button>
+        </div>
+      </div>
+      <Table
+        data={filteredProperties}
+        identifierMapping={dataObject => dataObject.id}
+        rowMappingToCells={property => [
+          (<Tile
+            key="cell-1"
+            title={{ value: property.basicInfo.propertyName }}
+            description={{ value: translation[property.field.fieldType] }}
+          />),
+          (<div key="cell-2" className={tw('flex flex-row gap-x-2')}>
+            <SubjectTypeIcon subjectType={property.basicInfo.subjectType}/>
+            <Span>{translation[property.basicInfo.subjectType]}</Span>
+          </div>),
+          (<div key="cell-3" className={tw('flex flex-row justify-end')}>
```

Comment:
```suggestion
          (<div key="edit-button-cell" className={tw('flex flex-row justify-end')}>
```
Do we really need all these keys?